tests/test_handlers.py

```diff
@@ -0,0 +1,172 @@
+                                 '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ '
+                                 'FRONTIER_START   info message{reset}\n'.format(
+                                     bold_yellow=re.escape(colors['bold_yellow']),
+                                     reset=re.escape(colors['reset'])))
+
+    def test_handler_events_level(self):
+        self.logger.debug('debug message', extra={'event': 'FRONTIER_START'})
+        self.logger.info('info message', extra={'event': 'FRONTIER_STOP'})
+        self.assertRegexpMatches(self.logger_output.getvalue(),
+                                 '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ '
+                                 'FRONTIER_STOP    info message{reset}\n'.format(
+                                     bold_yellow=re.escape(colors['bold_yellow']),
+                                     reset=re.escape(colors['reset'])))
+
+    def test_handler_events_filter(self):
+        self.logger.info({'message1': 'info', 'message2': 'message', 'event': 'FRONTIER_START'})
+        log = self.logger_output.getvalue()
+        pattern1 = '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ ' \
+                   'FRONTIER_START   {logged_message}{reset}\n'.format(bold_yellow=re.escape(colors['bold_yellow']),
+                                                                       logged_message='message info',
+                                                                       reset=re.escape(colors['reset']))
+        pattern2 = '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ ' \
+                   'FRONTIER_START   {logged_message}{reset}\n'.format(bold_yellow=re.escape(colors['bold_yellow']),
+                                                                       logged_message='info message',
+                                                                       reset=re.escape(colors['reset']))
+        assert re.match(pattern1, log) is not None or re.match(pattern2, log) is not None
+
+
+class TestHandlerConsole(BaseTestHandlers):
+    handler = CONSOLE
+
+    def test_handler_color_based_on_level(self):
+        self.logger.debug('debug message')
+        self.logger.info('info message')
+        self.logger.error('error message')
+        self.assertEqual(self.logger_output.getvalue(),
+                         '{white}[frontera] debug message{reset}\n'
+                         '{green}[frontera] info message{reset}\n'
+                         '{red}[frontera] error message{reset}\n'.format(white=colors['white'],
+                                                                         green=colors['green'],
+                                                                         red=colors['red'],
+                                                                         reset=colors['reset']))
+
+
+class TestHandlerConsoleManager(TestHandlerConsole):
```

Comment:
Ok, so let's remove _MANAGER, _BACKEND, _DEBUGGING and _EVENTS handlers and Redis-related stuff.